src/test/java/teammates/test/cases/action/InstructorStudentRecordsAjaxPageActionTest.java

```diff
@@ -78,10 +78,18 @@ protected InstructorStudentRecordsAjaxPageAction getAction(String... params) {
         return (InstructorStudentRecordsAjaxPageAction) gaeSimulation.getActionObject(getActionUri(), params);
     }
 
-    @Override
     @Test
+    @Override
     protected void testAccessControl() throws Exception {
-      //TODO: implement this
+        InstructorAttributes instructor = typicalBundle.instructors.get("instructor3OfCourse1");
+        StudentAttributes student = typicalBundle.students.get("student2InCourse1");
+
+        String[] submissionParams = new String[] {
+                Const.ParamsNames.COURSE_ID, instructor.courseId,
+                Const.ParamsNames.STUDENT_EMAIL, student.email,
+                Const.ParamsNames.FEEDBACK_SESSION_NAME, "First feedback session"
+        };
+        verifyOnlyInstructorsOfTheSameCourseCanAccess(submissionParams);
```

Comment:
Add `verifyUnaccessibleWithoutViewStudentInSectionsPrivilege`.

Reply:
This check is failing with `java.lang.AssertionError: expected:<You are not registered in the course > but was:<>`

Reply:
Can you add a test case to ensure that resultTables of InstructorStudentRecordsAjaxPageData is empty for the helper instructor?

Reply:
Actually, the test case I wanted you to add is above in testExecuteAndPostProcess, maybe you could move that test case here instead.

Reply:
Done